app/javascript/packs/leafletMap.js

```diff
@@ -31,7 +32,7 @@ export async function setupMap() {
 
     L.tileLayer("http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
         attribution:
-            '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a>',
+            '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> | Crosshairs icon by <a href="https://fontawesome.com/license/free">Font Awesome</a>',
```

Comment:
Is this really necessary? Otherwise we might design our own icon?

Reply:
No, actually it isn't. I overlooked the part of the [license page](https://fontawesome.com/license/free) where it says:

> Downloaded Font Awesome Free files already contain embedded comments with sufficient attribution, so you shouldn't need to do anything additional when using these files normally.

tldr: you can remove this.